recipes/dectate/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "dectate" %}
+{% set version = "0.13" %}
+{% set sha256 = "299a5d3d674d7cd095c8489331ecece22e5a567ee8a7636e8b57bbb220c568e4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  preserve_egg_dir: True
```

Comment:
I believe that the `preserve_egg_dir` should be deleted, although I'm not 100% sure. I have not seen it for any other recipes as far as I can remember.